pkg/controller/servicebinding/servicebinder_test.go

```diff
@@ -513,6 +525,10 @@ func TestServiceBinder_Bind(t *testing.T) {
 				Reason:  EmptyApplicationReason,
 				Message: errEmptyApplication.Error(),
 			},
+			{
+				Type:   BindingReady,
+				Status: corev1.ConditionTrue,
```

Comment:
According to https://github.com/redhat-developer/service-binding-operator/issues/665#issue-694667113 CollectionReady : true and InjectionReady : False gives Ready : False . @qibobo Please let me know if I am missing something here

Reply:
Because in https://github.com/redhat-developer/service-binding-operator/issues/665#issuecomment-692596417 I see that 2nd row and 4th row (last row) have collectionReady :true InjectionReady : false but Ready is both True and False for 2 similar granularities. Idk what was eventually agreed upon(there are 2 tables demonstrated in the issue). We need to look into that and accordingly change and also update the comment here as well https://github.com/redhat-developer/service-binding-operator/pull/681/files#diff-c9af90357b70995ba06d05f3e039ec5bed01449ec59b7318238804f523ba6364R24

Reply:
@Avni-Sharma 
CollectionReady=True, InjectionReady=False with emptyAppSelector, then Ready is true as there in app to do bind, so the whole process is successful.
CollectionReady=True, InjectionReady=False with appNotFound, then Ready is false because there is error to do injection, so the whole process is failed.

Reply:
Thanks @qibobo  . In that case, can we pls also have the appNotFound reason test case as well. wherein we show CollectionReady = True, InjectionRedy = false with appNotFound and then show `ready` as false